using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details.md

```diff
@@ -0,0 +1,83 @@
+# Rules engine consequence detail definition
+
+The Adobe Experience Platform Mobile SDK supports multiple types of rule consequences (also known as actions) that can be executed when the trigger event and the conditions are met. This section contains a detailed description of each rule consequence.
+
+## Analytics consequence
+
+This rule consequence is currently handled by the [Analytics](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics) extension.
+
+| Friendly name   | Key         | Type   | Description                                                  |
+| --------------- | ----------- | ------ | ------------------------------------------------------------ |
+| Action          | action      | string | *(Optional)* If provided, this value will be used as the action parameter in a `trackAction` call. |
+| State/Page name | state       | string | *(Optional)* If provided, this value will be used as the state parameter in a `trackState` call. |
+| Context data    | contextdata | object | *(Optional)* Additional context data to be attached to the resulting Analytics request. The object should only have one level of depth, that contains <string, string> key-value pairs. |
+
+## In-App message consequence
+
+This rule consequence is currently handled by the [Campaign](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaign-standard) extension.
+
+| Friendly name              | Key          | Type   | Description                                                  |
+| -------------------------- | ------------ | ------ | ------------------------------------------------------------ |
+| In-App template            | template     | string | *(Required)* Determines the display method of the In-App message. Currently supported types are `fullscreen`, `alert` and `local`. |
+| Fullscreen message content | html         | string | *(Required for fullscreen message type)* Filename of the HTML content to load for a fullscreen In-App message. This file will be sourced from the same zip file the `assets/` directory from which the consequence was retrieved. The base path (`assets/`) should not be included in this value. |
+| Remote assets              | remoteAssets | array  | *(Optional, only used by the fullscreen message type)* Array of strings of remote assets that should be cached by the client. These should be full-url strings with HTTPS transport. |
```

Comment:
In the 2nd description sentence,  please change "full-url" be changed to "full-URL"?